abci/cmd/abci-cli/abci-cli.go

```diff
@@ -642,7 +642,9 @@ func cmdCounter(cmd *cobra.Command, args []string) error {
 	// Stop upon receiving SIGTERM or CTRL-C.
 	tmos.TrapSignal(logger, func() {
 		// Cleanup
-		srv.Stop()
+		if err := srv.Stop(); err != nil {
+			panic(err)
```

Comment:
don't think we should panic here

Reply:
I'd rather see `_ = srv.Stop()` here or maybe `err := ..; if err != nil { logger.Error() }`